backend/src/main/java/gov/cdc/usds/simplereport/api/apiuser/UserAccountCreationController.java

```diff
@@ -39,19 +60,26 @@ private void init() {
   }
 
   /**
-   * Validates that the requesting user has been sent an invitation to SimpleReport, ensures the
-   * given password meets all requirements, and sets the password in Okta.
+   * Fetches a user status's status within the account creation process. Does not exactly match the
+   * Okta account status options, since we also want to know about states such as MFA enrollment.
    *
-   * @param requestBody contains the password
-   * @param request contains all header information, including the activation token.
-   * @throws InvalidActivationLinkException if the activation token is invalid.
-   * @throws OktaAuthenticationFailureException if the password is invalid or if the user is not in
-   *     a RESET_PASSWORD state.
+   * @param activationToken optional, the activationToken provided to a user
+   * @param userId optional, the Okta user id on a given session
+   * @param factorId optional, the Okta factor id on a given session
+   * @return UserAccountStatus
    */
-  @PostMapping("/initialize-and-set-password")
```

Comment:
That's a very good point, it's not backwards-compatible with the UI at all. However, since this hasn't been rolled out to users yet, I don't think there's a danger of bugs here since they _shouldn't_ be interacting with this UI at all yet. Good flag though, I'll keep an eye out for this kind of bug when we deploy.

Reply:
Ahh perfect. I didn't realize this hadn't gone live yet. Thanks!